### PR TITLE
Adding option to turn off the watchers

### PR DESCRIPTION
**Pull request by ggsood**
The change addresses issue #66 
Adding option to turn off the watchers
- Adding an option variable "active" which when set to false will deactivate the watcher.

### CODE REVIEW

**Comment by ggsood** [comment]
@phillbaker please can you check the change.

File: es/resource_elasticsearch_xpack_watch.go [hunk 1]
```diff
@@ -198,5 +205,34 @@ func resourceElasticsearchPutWatch(d *schema.ResourceData, m interface{}) (strin
 		return "", err
 	}
 
+	activateWatcher(esClient, watchID, isActive)
```
Comment (phillbaker) [inline es/resource_elasticsearch_xpack_watch.go:208]: Looks like the lint failed b/c the error returned here is not checked.

**Review by phillbaker — APPROVED** [review]
This looks good to me, just need to fix the linting failures.

Can you also add a line to the CHANGELOG please?

**Comment by ggsood** [comment]
Sure will update it.

**Comment by ggsood** [comment]
hi @phillbaker have updated the code and added the changelog, please can you check.

**Comment by ggsood** [comment]
thanks @phillbaker waiting for the PR to be merged :)

**Comment by phillbaker** [comment]
@ggsood one more ask here, we need to update `resourceElasticsearchWatchRead` to read this information as well and set it in the `active` field. That will allow importing watches that are deactivated in the correct state.

**Comment by ggsood** [comment]
Thanks @phillbaker  I will work on it now

**Comment by ggsood** [comment]
Hi @phillbaker  the change is now done, and tested.

**Pull request merged**